OMEdit/OMEdit/OMEditGUI/Annotations/ShapePropertiesDialog.cpp

```diff
@@ -201,10 +201,28 @@ ShapePropertiesDialog::ShapePropertiesDialog(ShapeAnnotation *pShapeAnnotation,
     mpFontNameComboBox->setCurrentIndex(0);
   }
   mpFontSizeLabel = new Label(Helper::size);
-  mpFontSizeSpinBox = new DoubleSpinBox;
-  mpFontSizeSpinBox->setRange(0, std::numeric_limits<double>::max());
-  mpFontSizeSpinBox->setValue(mpShapeAnnotation->getFontSize());
-  mpFontSizeSpinBox->setSingleStep(1);
+  mpFontSizeComboBox = new QComboBox;
+  mpFontSizeComboBox->setEditable(true);
+  // get the standard font sizes
+  QList<int> standardFontSizes = QFontDatabase::standardSizes();
```

Comment:
If I am to be picky this should be a const, since we never change the reference but only the object inside it

Reply:
I don't think its possible. We actually change the list so it can't be `const`.